powerhome/playbook
# [PLAY-236] Rebuild Charts to Use React-Highcharts

### This PR removes the custom pbChart plugin that was being used for the Highcharts based kits and instead rebuilds those kits to use React-Highcharts wrapper. Kits effected:

- Bar Graph
- Line Graph
- Circle Chart
- Treemap Chart
- Gauge Chart

The above kits have also been converted to .tsx
#### Screens

<img width="1245" alt="Screenshot 2022-11-30 at 8 48 32 AM" src="https://user-images.githubusercontent.com/73710701/204817288-0f1a5ef6-ecf3-4a57-859e-c908ff5bca6c.png">

<img width="1236" alt="Screenshot 2022-11-30 at 9 02 22 AM" src="https://user-images.githubusercontent.com/73710701/204817329-9d2e5b8a-1182-491c-b702-c6c56953248d.png">

<img width="1251" alt="Screenshot 2022-11-30 at 9 02 38 AM" src="https://user-images.githubusercontent.com/73710701/204817347-0b3d1c92-e8b9-44b8-acf4-8ca782759f15.png">

<img width="1215" alt="Screenshot 2022-11-30 at 9 02 55 AM" src="https://user-images.githubusercontent.com/73710701/204817415-6852f8ee-a7c4-436e-b86c-a95d3ad6f74f.png">

<img width="1255" alt="Screenshot 2022-11-30 at 9 03 09 AM" src="https://user-images.githubusercontent.com/73710701/204817435-e7c1f56c-44b2-4d60-a353-5becf341e237.png">

#### Breaking Changes

Yes, removes pbChart plugin and refactors charts

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-236)

#### How to test this

To test this story:

- Check each of the above mentioned charts to make sure all props work as expected.
- Check react as well as rails versions of the kits
- Compare charts in the milano env to prod to make sure nothing has changed visually
